lua/astrocommunity/editing-support/refactoring-nvim/refactoring-nvim.lua

```diff
@@ -3,4 +3,97 @@ return {
   dependencies = { "nvim-lua/plenary.nvim", "nvim-treesitter/nvim-treesitter" },
   event = "User AstroFile",
   opts = {},
+  keys = {
+    {
+      "<leader>re",
+      [[ <Esc><Cmd>lua require('refactoring').refactor('Extract Function')<CR>]],
+      { silent = true, expr = false },
+      mode = {
+        "v",
+      },
+      desc = "Extract Function",
+    },
+    {
+      "<leader>rf",
+      [[ <Esc><Cmd>lua require('refactoring').refactor('Extract Function To File')<CR>]],
+      { silent = true, expr = false },
+      mode = {
+        "v",
+      },
+      desc = "Extract Function To File",
+    },
+    {
+      "<leader>rv",
+      [[ <Esc><Cmd>lua require('refactoring').refactor('Extract Variable')<CR>]],
+      { silent = true, expr = false },
+      mode = {
+        "v",
+      },
+      desc = "Extract Variable",
+    },
+    {
+      "<leader>ri",
+      [[ <Esc><Cmd>lua require('refactoring').refactor('Inline Variable')<CR>]],
+      { silent = true, expr = false },
+      mode = {
+        "v",
+      },
+      desc = "Inline Variable",
+    },
+    {
+      "<leader>rb",
+      [[ <Cmd>lua require('refactoring').refactor('Extract Block')<CR>]],
+      { silent = true, expr = false },
+      mode = {
+        "n",
+      },
+      desc = "Extract Block",
+    },
+    {
+      "<leader>rbf",
+      [[ <Cmd>lua require('refactoring').refactor('Extract Block To File')<CR>]],
+      { silent = true, expr = false },
+      mode = {
+        "n",
+      },
+      desc = "Extract Block To File",
+    },
+    {
+      "<leader>ri",
+      [[ <Cmd>lua require('refactoring').refactor('Inline Variable')<CR>]],
+      { silent = true, expr = false },
+      mode = { "n" },
+      desc = "Inline Variable",
+    },
```

Comment:
```suggestion
    {
      "<leader>rb",
      function() require('refactoring').refactor('Extract Block') end,
      { silent = true, expr = false },
      mode = {
        "n",
      },
      desc = "Extract Block",
    },
    {
      "<leader>rbf",
      function() require('refactoring').refactor('Extract Block To File') end,
      { silent = true, expr = false },
      mode = {
        "n",
      },
      desc = "Extract Block To File",
    },
    {
      "<leader>ri",
      function() require('refactoring').refactor('Inline Variable') end,
      { silent = true, expr = false },
      mode = { "n" },
      desc = "Inline Variable",
    },
```

These should just be lua functions

Reply:
Sure, I mostly just copied pasted